### DISCUSSION TITLE
CPU usage is too high

### DISCUSSION THREAD

**Discussion by cctv130** [rustdesk/rustdesk#7893] [Q&A]
![image](https://github.com/rustdesk/rustdesk/assets/133784576/e0771ced-5082-45f3-955e-40d3a799cf05)

**Comment by xlionjuan** [comment]
It will if on low-end CPU, and using software rendering

**Comment by dimejo** [comment]
@cctv130:
Which codec are you using? Have you tried using a different codec?

**Reply by cctv130** [reply]
I now use the windows platform and now it performs well
